### PR TITLE
Import ScrollContainer component and fix plugin path

### PR DESCRIPTION
**Pull request by skyroses**
## Description
Added a forgotten import of the ScrollContainer component and fix path to the plugin 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/nuxt/src/composables/use-plugin.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import type { VuesticOptions } from './../types';
 /** Register vuestic nuxt plugin */
 export const useVuesticPlugin = (options: VuesticOptions) => {
   addPluginTemplate({
-    src: resolve(distDir, './runtime/plugin.mjs'),
+    src: resolve(distDir, './runtime/plugin.ts'),
```
Comment (m0ksem) [inline packages/nuxt/src/composables/use-plugin.ts:11]: Must stay `.mjs`, cuz `ts` file is compiled to `mjs`.

Were there any issues with that before?

**Review by m0ksem — CHANGES_REQUESTED** [review]

File: packages/nuxt/src/composables/use-plugin.ts (hunk 1 above)
Comment (skyroses) [reply packages/nuxt/src/composables/use-plugin.ts:11]: Without this, the module is not compiled

Comment (aluarius) [reply packages/nuxt/src/composables/use-plugin.ts:11]: It's compiled and with `.ts` here will be a wrong path (dist module.mjs on the screen):

image

**Review by aluarius — CHANGES_REQUESTED** [review]

File: packages/nuxt/src/composables/use-plugin.ts (hunk 1 above)
Comment (skyroses) [reply packages/nuxt/src/composables/use-plugin.ts:11]: I didn't change the filename. 
With src dir (`resolve(distDir, "./runtime/plugin.mjs")`) we can't get types using `dev:prepare`. We get an error that there is no such file. With an extension .ts everything is fine

Comment (skyroses) [reply packages/nuxt/src/composables/use-plugin.ts:11]: And the `filename` option specifies how the file will be named after compilation - there should be `mjs`, yes

**Comment by m0ksem** [comment]
Moved to https://github.com/epicmaxco/vuestic-ui/pull/2610

**Pull request closed without merging**